variables.tf

```diff
@@ -69,35 +69,35 @@ variable "existing_machine_learning_instance_crn" {
   type        = string
 }
 
-variable "watsonx_machine_learning_instance_name" {
+variable "watsonx_ai_runtime_instance_name" {
   type        = string
   description = "The name of the Watson Machine Learning instance to create. If a prefix input variable is passed, it is prefixed to the value in the `<prefix>-value` format."
   default     = "watsonx-ml"
 }
 
-variable "watson_machine_learning_plan" {
+variable "watsonx_ai_runtime_plan" {
   description = "The plan that is used to provision the Watson Machine Learning instance. Allowed values are 'lite', 'v2-professional' and 'v2-standard'."
   type        = string
   default     = "lite"
 
   validation {
-    condition     = contains(["lite", "v2-professional", "v2-standard"], var.watson_machine_learning_plan)
+    condition     = contains(["lite", "v2-professional", "v2-standard"], var.watsonx_ai_runtime_plan)
     error_message = "The plan must be lite, v2-professional, or v2-standard for watsonx.ai Runtime."
   }
 }
 
-variable "watson_machine_learning_service_endpoints" {
+variable "watsonx_ai_runtime_service_endpoints" {
   type        = string
   description = "The type of service endpoints. Possible values: 'public', 'private', 'public-and-private'."
```

Comment:
Must clarify the description to tell the endpoints belong to which service. `The type of service endpoints for watsonx.ai Runtime. ..`